pandas/core/series.py

```diff
@@ -4990,6 +4990,24 @@ def _logical_method(self, other, op):
         res_values = ops.logical_op(lvalues, rvalues, op)
         return self._construct_result(res_values, name=res_name)
 
+    def _arith_method(self, other, op):
+        res_name = ops.get_op_result_name(self, other)
+        self, other = ops.align_method_SERIES(self, other)
+
+        lvalues = extract_array(self, extract_numpy=True)
+        rvalues = extract_array(other, extract_numpy=True)
+        result = ops.arithmetic_op(lvalues, rvalues, op)
+
+        return self._construct_result(result, name=res_name)
+
+    def __div__(self, other):
+        # Alias for backward compat
+        return self.__truediv__(other)
+
+    def __rdiv__(self, other):
+        # Alias for backward compat
+        return self.__rtruediv__(other)
```

Comment:
any objection to just ripping out these `__div__` and `__rdiv__` aliases?  how about the flex versions `div` and `rdiv`?

Reply:
nope

these are no longer needed in py3 iirc